server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -163,7 +166,7 @@ module.exports = class ReportUploadRequestHandler {
     return productMetaProperties;
   }
 
-  async jsonSave(repository, data, productMetaData) {
+  async jsonSave(repository: any, data: any, productMetaData: any): Promise<void> {
```

Comment:
Can you make those types more precise?

Reply:
yeah I will define a report types for it, also I will refactor function below to use deconstruction